tools/monorepo-utils/src/changelog/index.ts

```diff
@@ -0,0 +1,199 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+import { execSync } from 'child_process';
+import simpleGit from 'simple-git';
+import nodePath from 'path';
+import { existsSync } from 'fs';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../core/logger';
+import { isGithubCI } from '../core/environment';
+import { cloneAuthenticatedRepo, checkoutRemoteBranch } from '../core/git';
+import {
+	getPullRequestData,
+	getShouldAutomateChangelog,
+	getChangelogDetails,
+} from './lib/github';
+import {
+	getAllProjectPaths,
+	getTouchedProjectsRequiringChangelog,
+} from './lib/projects';
+
+const program = new Command( 'changelog' )
```

Comment:
For now it's focused on automating changlog creation from pull request details, so something more specific might be easier to understand:

```suggestion
const program = new Command( 'changelog-from-pr' )
```

Having to type more won't matter as much as it'll usually run from a CI context.

Reply:
In general, I think we misuse the term "changelog" 😄.

* "changelog" / "changelog file": the compiled list of all changes in a release.
* "change file": a file containing info about a single change tied to a PR.

The latter is what we're generating here... change files, not changelog files.

Reply:
How about `pnpm utils changelog file 8988` as a command?

Reply:
`pnpm utils changefile 8988`?